xml/System/Type.xml

```diff
@@ -7160,7 +7160,7 @@ If the current <xref:System.Type> represents a type parameter in the definition
 ## Remarks  
  The search for nested types is not recursive.  
   
- The <xref:System.Type.GetNestedTypes%2A> method does not return types in a particular order, such as alphabetical or declaration order. Your code must not depend on the order in which types are returned, because that order varies.  
+ In .NET 6 and below versions the <xref:System.Type.GetNestedTypes%2A> method does not return types in a particular order, such as alphabetical or declaration order. Your code must not depend on the order in which types are returned, because that order varies. However, starting with .NET 7, the ordering is deterministic based upon the metadata ordering in the assembly.
```

Comment:
```suggestion
 In .NET 6 and earlier versions, the <xref:System.Type.GetNestedTypes%2A> method does not return types in a particular order, such as alphabetical or declaration order. Your code must not depend on the order in which types are returned, because that order varies. However, starting with .NET 7, the ordering is deterministic based upon the metadata ordering in the assembly.
```